doc/api/readline.md

```diff
@@ -90,7 +90,7 @@ The `'pause'` event is emitted when one of the following occur:
 
 * The `input` stream is paused.
 * The `input` stream is not paused and receives the `SIGCONT` event. (See
-  events `SIGTSTP` and `SIGCONT`)
+  events [SIGTSTP][] and [SIGCONT][])
```

Comment:
Hm, these should probably keep their `` ` ``? I think the signal names are marked as code everywhere else here.